src/components/IndividualArticle.jsx

```diff
@@ -1,15 +1,32 @@
 import React from "react";
 import { useState, useEffect } from 'react'
-import { getArticleById, getCommentsByArticle } from "./api"
+import { getArticleById, getCommentsByArticle, updateArticleVotes } from "./api"
 import { useParams } from "react-router-dom"
 import CommentCard from "./CommentCard"
 
 const IndividualArticle = () => {
     const [article, setArticle] = useState({})
     const [comments, setComments] = useState([])
     const [loading, setLoading] = useState(true)
+    const [userVotes, setUserVotes] = useState(0);
     const { article_id } = useParams();
+    const [disabled, setDisabled] = useState(false);
 
+    function increaseVote(e) {
+        e.preventDefault()
+        const updateVotesBy = { inc_votes: 1 }
+        setUserVotes(1)
+        updateArticleVotes(article_id, updateVotesBy)
```

Comment:
What if this patch (or the patch below) fails? We would need to undo the optimistic rendering and also give the client some sort of feedback that there was an error